jetbrains-core/src/software/aws/toolkits/jetbrains/core/credentials/SsoSessionConfigurationManager.kt

```diff
@@ -57,7 +58,7 @@ class SsoSessionConfigurationManager {
 
         if (ssoSessionSection?.isEmpty == false) {
             val existing = """
-            [$SSO_SESSION_SECTION_NAME $ssoProfileName]
+            [$SSO_SESSION_SECTION_NAME $idcProfileName]
```

Comment:
shouldn't this stay as `ssoProfileName`?

Reply:
Yes it is, it has changed due to a typo in refactoring. Will revert this